core/genesis.go

```diff
@@ -108,11 +120,21 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, genesis *genesisT.Genesis,
 	// Get the existing chain configuration.
 	newcfg := configOrDefault(genesis, stored)
 
-	// NOTE(ia): does not include overrideBerlin
-
-	if ecbp1100 != nil {
-		n := ecbp1100.Uint64()
-		if err := newcfg.SetECBP1100Transition(&n); err != nil {
+	if overrideMagneto != nil {
+		n := overrideMagneto.Uint64()
+		if err := newcfg.SetEIP2565Transition(&n); err != nil {
+			return newcfg, stored, err
+		}
+		if err := newcfg.SetEIP2315Transition(&n); err != nil {
```

Comment:
`EIP2315 Simple Subroutines` should be removed. There's a PR in doing this that hasn't been merged yet. https://github.com/ethereumclassic/ECIPs/pull/419/files.

Reply:
Done with https://github.com/etclabscore/core-geth/pull/367/commits/af67af42d0df129231eb81686689b47fc6530c3e